addons/survey/tests/test_survey.py

```diff
@@ -175,3 +173,106 @@ def test_result_data_numeric_box(self):
         result = self.env['survey.survey'].prepare_result(question)
         for key in exresult:
             self.assertEqual(result[key], exresult[key])
+
+    def test_get_input_summary_without_filters_and_no_answer(self):
+        question = self._add_question(self.page_0, 'Q1', 'numerical_box')
+        result = self.env['survey.survey'].get_input_summary(question)
+        self.assertDictEqual(result, {"answered": 0, "skipped": 0})
+
+    def test_get_input_summary_without_filters_and_simple_choice_answers_only(self):
+        question = self._add_question(
+            self.page_0, 'Q0', 'simple_choice',
+            labels=[{'value': 'Choice0'}, {'value': 'Choice1'}]
+        )
+
+        # 2 answered + 1 skipped
+        answers = [self._add_answer(self.survey, False) for i in range(3)]
+
+        self._add_answer_line(question, answers[0], random.choice(question.labels_ids.ids))
+        self._add_answer_line(question, answers[1], random.choice(question.labels_ids.ids))
+        self._add_answer_line(
+            question, answers[2], random.choice(question.labels_ids.ids),
```

Comment:
Strange to specify a "answer_value" for a skipped question, is it necessary?

Reply:
The parameter `answer_value` is mandatory so I have to provide a value. But, to be more explicit, I will use `None` instead of a real value.